### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Vadivel-Babu**
[DEMO LINK](https://Vadivel-Babu.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,18 @@
...
+  const married = person.isMarried
+    ? `${person.partnerName} is my ${sex}`
+    : 'I am not married';
+  const name = `My name is ${person.name}`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{name}</h2>
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+      <p className="Person__partner">{married}</p>
```
Comment (polosanya) [inline src/components/Person/Person.jsx:15]:
```suggestion
      <h2 className="Person__name">{name}</h2>
      
      {person.age 
        && <p className="Person__age">{`I am ${person.age}`}</p>}
      
      <p className="Person__partner">{married}</p>
```

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,18 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
```
Comment (polosanya) [inline src/components/Person/Person.jsx:4]: image

File: src/components/Person/Person.jsx [hunk 3]
```diff
@@ -1 +1,18 @@
...
+  const sex = person.sex === 'm' ? 'wife' : 'husband';
+  const married = person.isMarried
+    ? `${person.partnerName} is my ${sex}`
+    : 'I am not married';
+  const name = `My name is ${person.name}`;
```
Comment (polosanya) [inline src/components/Person/Person.jsx:9]: I'm not sure if you need this variable. You can add this string directly

**Review by polosanya — CHANGES_REQUESTED** [review]
Please don't forget to add a link to your PR description
image

File: src/components/Person/Person.jsx [hunk 4]
```diff
@@ -1 +1,18 @@
...
+  const Sex = sex === 'm' ? 'wife' : 'husband';
+  const married = isMarried
+    ? `${partnerName} is my ${Sex}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">{married}</p>
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/components/Person/Person.jsx:15]: Add divided empty lines as it was said in previous mentor's comment

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Hi! Seems like you didn't fix all previous comments, check them again please

File: src/components/Person/Person.jsx [hunk 5]
```diff
@@ -1 +1,17 @@
 // export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, sex, age, isMarried, partnerName } = person;
```
Comment (polosanya) [inline src/components/Person/Person.jsx:5]:
```suggestion
  const { 
    name, 
    sex, 
    age, 
    isMarried, 
    partnerName,
  } = person;
```

File: src/components/Person/Person.jsx [hunk 6]
```diff
@@ -1 +1,17 @@
...
+  const Sex = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">
```
Comment (polosanya) [inline src/components/Person/Person.jsx:12]:
```suggestion
      <h2 className="Person__name">{`My name is ${name}`}</h2>
      
      {age && <p className="Person__age">{`I am ${age}`}</p>}
      
      <p className="Person__partner">
```

File: src/components/Person/Person.jsx [hunk 7]
```diff
@@ -1 +1,17 @@
...
+        {isMarried ? `${partnerName} is my ${Sex}` : 'I am not married'}
```
Comment (polosanya) [inline src/components/Person/Person.jsx:13]:
```suggestion
        {isMarried 
          ? `${partnerName} is my ${Sex}` 
          : 'I am not married'}
```

**Review by polosanya — CHANGES_REQUESTED** [review]
Let's polish it up) 
Don't forget to push the changes
Feel free to ask for help in the Slack chat if you have any difficulties)

**Comment by Vadivel-Babu** [comment]
I formatted the document as per you said but if I save the document after change the format it went back to single line.

**Review by Volodymyr-Melnyk-Fron10der — APPROVED** [review]
Good then👍

**Pull request closed without merging**